suite-native/module-home/src/components/DashboardHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Icon } from '@trezor/icons';
+import { Box } from '@suite-native/atoms';
+
+import { DashboardHeaderDeviceChip } from './DashboardHeaderDeviceChip';
+
+const headerStyle = prepareNativeStyle(() => ({
+    width: '100%',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+}));
+
+export const DashboardHeader = () => {
+    const { applyStyle } = useNativeStyles();
+    return (
+        <Box style={applyStyle(headerStyle)}>
+            <DashboardHeaderDeviceChip />
+            <Box flexDirection="row">
+                <Box marginRight="medium">
+                    <Icon name="eyeSlash" />
```

Comment:
This was added in Figma. Ondra said he still needs to decide whether they're going to be hollow or filled so this will be changed later so they're both same.